### PR TITLE
Hello Marinenko

### PR DESCRIPTION
**Pull request by MaxPanzer**
London is the capital of Great Britain on river Thames

### CODE REVIEW

File: MarinenkoFoo/ViewController.swift [hunk 1]
```diff
@@ -11,7 +11,9 @@ class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        print("Hi")
```
Comment (AndreySlimus) [inline MarinenkoFoo/ViewController.swift:14]: max loh

Comment (AndreySlimus) [reply MarinenkoFoo/ViewController.swift]: loh

Comment (MaxPanzer) [reply MarinenkoFoo/ViewController.swift:14]: max krasav4ik

Comment (AndreySlimus) [reply MarinenkoFoo/ViewController.swift]: ok, ispravleno

File: MarinenkoFoo/AppDelegate.swift [hunk 2]
```diff
@@ -13,7 +13,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        // Override point for customization after application launch.
+        print("dwdwdadx")
```
Comment (AndreySlimus) [inline MarinenkoFoo/AppDelegate.swift:16]: ffbhydbhy

File: MarinenkoFoo/AppDelegate.swift [hunk 3]
```diff
@@ -13,7 +13,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+        print("yjhjy")
+        print("65758h")
```
Comment (AndreySlimus) [inline MarinenkoFoo/AppDelegate.swift:18]: rdghbgrdhrt

File: MarinenkoFoo/ViewController.swift [hunk 4]
```diff
@@ -11,7 +11,7 @@ class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        print("Changed")
```
Comment (AndreySlimus) [inline MarinenkoFoo/ViewController.swift:14]: good

**Pull request merged**